Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -107,6 +117,36 @@ private static void CreateInputActionReferences(InputActionAsset asset)
             }
         }
 
+        internal static void CheckForDefaultUIActionMapChanges()
+        {
+            var asset = GetOrCreate();
+            if (asset != null)
+            {
+                if (s_DefaultUIMap == null)
+                    GetDefaultUIActionMap();
+                
+                var uiMap = asset.actionMaps.FirstOrDefault(m => m.name == "UI");
+                // "UI" action map has been removed or renamed.
+                if (uiMap == null)
+                {
+                    Debug.LogWarning("The default UI action map does not exist.\r\n " +
+                        "This will break the UI input at runtime. Please revert the changes to restore the default UI action map.");
+                    return;
+                }
+
+                // "UI" action map has been modified.
+                foreach (var action in s_DefaultUIMap)
+                {
+                    if(uiMap.FindAction(action.name) == null)
+                    {
+                        Debug.LogWarning($"The UI action \"{action.name}\" has been modified.\r\n " +
```

Comment:
Similar to previous comment on warning, instead of "modified" shouldn't this warning instead state that it "is missing" or "do not exist." The key isn't to restore or revert but rather to provide an action with the expected name/type?